environments/prod/prod.tfvars

```diff
@@ -1007,6 +1017,26 @@ frontends = [
         operator       = "Equals"
         selector       = "rf"
       },
+      {
```

Comment:
Minor one: could you please place cookie and req param exclusions in one place and ideally in alphabetical order or selectors, for better readability/maintainability?

Reply:
yeah sure, that will make it more readable ;)